src/components/templates/AddStory.js

```diff
@@ -31,14 +34,15 @@ const AddStory = ({ toggleModal }) => {
         .add({
           title: newStory.title,
           content: newStory.content,
+          tags: [...tags],
```

Comment:
we don't need to create a new array when we're just passing the value over to our db
```suggestion
          tags: tags,
```